flair/embeddings/token.py

```diff
@@ -260,6 +269,93 @@ def extra_repr(self):
 
         return f"'{self.embeddings}'"
 
+class POSEmbeddings(TokenEmbeddings):
```

Comment:
@whoisjones perhaps you could remove the POS embeddings part from this PR, and add them in a separate PR?